packages/salesforcedx-vscode-core/src/orgBrowser/metadataOutlineProvider.ts

```diff
@@ -84,10 +84,10 @@ export class MetadataOutlineProvider
         break;
       case NodeType.Folder:
       case NodeType.MetadataType:
-        let nodeType = NodeType.MetadataComponent;
+        let nodeType: NodeType = NodeType.MetadataComponent;
         if (TypeUtils.FOLDER_TYPES.has(element.fullName)) {
           nodeType = NodeType.Folder;
-        } else if (element.type === NodeType.Folder && element.fullName) {
+        } else if (element.parent && element.parent.fullName === 'CustomObject') {
```

Comment:
nit: Can you make 'CustomObject' a constant? 
Also this logic change hurts my brain a little.  So previous
if it's a folder and has a fullName then it's a MetadataField
now
if it has a parent and it's parent has a full name that's equal to Custom Object then it's a MetadataField. 

Does this still work for not things in inside folders?

Reply:
Yes, rest everything listed under metadata type is set to the default i.e. MetadataComponent, be it within folder or without folders.